openlibrary/plugins/upstream/models.py

```diff
@@ -494,7 +494,11 @@ def get_covers(self, use_solr=True):
             return []
             
     def get_covers_from_solr(self):
+        try:
         w = self._solr_data
```

Comment:
It looks like the tabbing could be off here, has this been run w/ `pylint -E`? This is a convenient way (pre commit style procedure) to catch fun stuff like this)